### PR TITLE
chore(CX-2805): redesign profile ui

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [CX-2805] 

### Description

Redesign Profile UI

| **After** | **Before** |
| --- | ---|
| Simulator Screen Shot - iPhone 14 Pro - 2022-11-15 at 12 27 21 | Simulator Screen Shot - iPhone 14 Pro - 2022-11-15 at 12 29 10 |

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Update profile UI - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2805]: https://artsyproduct.atlassian.net/browse/CX-2805?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyProfile/MyProfileHeader.tsx [hunk 1]
```diff
@@ -71,10 +62,10 @@ export const MyProfileHeader: React.FC<{ me: MyProfileHeader_me$key }> = (props)
         <Touchable
           haptic
           hitSlop={{ top: 10, left: 10, right: 10, bottom: 10 }}
-          // @ts-expect-error
-          onPress={() => navigation.navigate("MyProfileEditForm")}
+          onPress={() => navigate("/my-profile/settings")}
+          style={{ height: "100%" }}
```
Comment (MrSltun) [inline src/app/Scenes/MyProfile/MyProfileHeader.tsx:66]: **nitpick:** The icon is not centered due to the height 👀

File: src/app/Scenes/MyProfile/MyProfileHeader.tsx [hunk 2]
```diff
@@ -27,36 +25,29 @@ export const MyProfileHeader: React.FC<{ me: MyProfileHeader_me$key }> = (props)
...
-          ) : (
-            <Image source={require("images/profile_placeholder_avatar.webp")} />
-          )}
+          <TouchableOpacity onPress={() => navigate("/my-profile/edit")}>
+            {!!userProfileImagePath ? (
+              <Avatar src={userProfileImagePath} size="xs" />
+            ) : (
+              <Image source={require("images/profile_placeholder_avatar.webp")} />
+            )}
+          </TouchableOpacity>
```
Comment (MrSltun) [inline src/app/Scenes/MyProfile/MyProfileHeader.tsx:50]: **Question: ** is it better to use `TouchableOpacity` here due to the element being an image? or is it the same as using `Touchable` from Palette?

**Review by MrSltun — COMMENTED** [review]
Looks good! Thank you Mounir 🌟  I just left a few comments :)

File: src/app/Scenes/MyProfile/MyProfileHeader.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Scenes/MyProfile/MyProfileHeader.tsx:66]: In the new designs, it's not supposed to be centered actually
Screenshot 2022-11-15 at 15 19 57

File: src/app/Scenes/MyProfile/MyProfileHeader.tsx (hunk 2 above)
Comment (MounirDhahri) [reply src/app/Scenes/MyProfile/MyProfileHeader.tsx:50]: The main reason here behind using the `TouchableOpacity` is to have some visual feedback for users to know they can press it. Using the `Touchable` component, the feedback is only there after you press

File: src/app/Scenes/MyProfile/MyProfileHeader.tsx (hunk 1 above)
Comment (MrSltun) [reply src/app/Scenes/MyProfile/MyProfileHeader.tsx:66]: Oh I didn't know that, I just checked the ticket and saw it centered in the picture, marking it as resolved 🙇

**Review by MrSltun — DISMISSED** [review]

**Comment by MounirDhahri** [comment]
working on fixing tests now 🔧🔧🔧

**Pull request merged**